incubator-tools/test_harness_guide/test_harness_guide.ipynb

```diff
@@ -0,0 +1,1455 @@
+{
```

Comment:
<p>Overall, this seems over complicated. This code can be simplified/made more modular and readable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>This comment taken-care in updated notebook</p>